server/documents/cursor.js

```diff
@@ -7,6 +7,18 @@ import randomColor from 'randomcolor';
 
 // Server-side only method, so we are not using ValidatedMethod.
 Meteor.methods({
+
+  'Cursor.remove'(args) {
+    check(args, {
+      contentKey: Match.DocumentId,
+      clientId: Match.DocumentId,
+    });
+    Cursor.documents.remove({
+      contentKey: args.contentKey,
+      clientId: args.clientId,
```

Comment:
I would add also `connectionId: this.connection.id` as a condition.

Reply:
Condition added.